tools/cldr-code/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -1692,7 +1694,20 @@ public void writePackageList(String outputDir) throws IOException {
             final String baseName = e.getKey();
 
             if (baseName.equals("IGNORE") || baseName.equals("cal")) continue;
-            if (baseName.equals("core") || baseName.equals("rbnf") || baseName.equals("bcp47")) {
+
+            boolean tiered = !baseName.equals("core");
+            // If it's a known un-tiered enum type, skip tiered.
+            try {
+                RunType r = RunType.valueOf(baseName);
+                if (!r.tiered()) {
+                    tiered = false;
+                }
+            } catch (IllegalArgumentException t) {
+                // ignored
+            }
+
+            // if not tiered
+            if (!tiered) {
```

Comment:
ideally there should be a subroutine `isTiered` to encapsulate this, and it would return false as soon as it saw "core". The surrounding method `writePackageList` is too long, over 100 lines; it's writing two files at once, cldr-packages.json and PACKAGES.md. It would be helpful -- make changes easier to review -- if it were organized into subroutines

Reply:
yes and part of the challenge is that it's not looking at RunType at that point but output directories.  But agree on subroutines.